test/manage/manage-bar.test.ts

```diff
@@ -45,6 +45,11 @@ describe('Manage bar', () => {
     expect(el.shadowRoot?.querySelector('.unselect-all-btn')).to.exist;
   });
 
+  it('have delete button disabled', async () => {
+    const el = await fixture<ManageBar>(html`<manage-bar></manage-bar>`);
+    expect(el.shadowRoot?.querySelector('.remove-btn:disabled')).to.exist;
+  });
```

Comment:
let's add another test here in this block when the `removeAllowed` prop is toggled to complete